third_party/move/tools/move-mutator/src/lib.rs

```diff
@@ -69,27 +69,22 @@ pub fn run_move_mutator(
 
         trace!("Processing file: {path:?}");
 
-        // Check if file is not excluded from mutant generation.
-        //TODO(asmie): refactor this when proper filtering will be introduced in the M3
-        if let Some(excluded) = mutator_configuration.project.exclude_files.as_ref() {
-            if excluded.contains(&path.to_path_buf()) {
-                continue;
+        // This `i` must be here as we must iterate over all mutants for a given file (ext and internal loop).
+        let mut i = 0;
```

Comment:
Well, since this is not a `for` loop, the comment above could be avoided with this name maybe
```suggestion
        let mut mutant_file_idx = 0;
```

Reply:
Done.